backend/src/schema/resolvers/todos.spec.js

```diff
@@ -16,6 +18,30 @@ const GET_TODOS = gql`
   }
 `;
 
+const GET_TODOS_PAGED = gql`
+  query {
```

Comment:
You could use arguments here:
```suggestion
  query($first: Int, $offset: Int) {
    todos(first: $first, offset: $offset) {
      // ...
    }
  }
```
and then pass a `variables` object to your call of `query` or `mutate` in your tests.